src/AI/mcp/server/tools/__init__.py

```
@@ -3,140 +3,345 @@
-def initialize_mcp():
-    """Initialize the MCP server."""
+def initialize_mcp(port: int = 8069):
+    """Initialize the MCP server with the specified port."""
     global mcp
     if mcp is not None:
         return mcp  # Already initialized
 
     mcp = FastMCP(
         name="altinity_mcp_server",
         instructions=ALTINITY_INSTRUCTIONS,
+        host="0.0.0.0",
```

Ruff (0.15.1)

[error] Possible binding to all interfaces

(S104)